x-pack/legacy/plugins/lens/public/indexpattern_plugin/indexpattern_suggestions.ts

```diff
@@ -49,10 +50,15 @@ function buildSuggestion({
       : state,
 
     table: {
-      columns,
-      isMultiRow: isMultiRow || columns.some(col => !columnMap[col.columnId].isMetric),
+      columns: columnOrder.map(columnId => ({
+        columnId,
+        operation: columnToOperation(columnMap[columnId]),
+      })),
+      isMultiRow: typeof isMultiRow === 'undefined' || isMultiRow,
```

Comment:
That's not what this check does - if `isMultiRow` is not defined, this will be true, otherwise it will be the value of `isMultiRow`. But I think this is highlighting that the statement is hard to parse. I removed `isMultiRow` and just look at the columns in `buildSuggestion`